src/test/java/net/sf/jabref/logic/exporter/BibTeXMLExporterTestFiles.java

```diff
@@ -45,23 +48,32 @@
 
     @Parameter
     public String filename;
-    public Path resourceDir;
+    public URI resource;
 
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
+    private Path resourceDir;
 
 
     @Parameters(name = "{0}")
     public static Collection<String> fileNames() throws IOException, URISyntaxException {
-        try (Stream<Path> stream = Files.list(Paths.get(BibTeXMLExporterTestFiles.class.getResource("").toURI()))) {
+        String packageName = BibTeXMLExporterTestFiles.class.getPackage().getName();
+        Path path = convertPackageNameToPath(packageName);
+
+        try (Stream<Path> stream = Files.list(path)) {
             return stream.map(n -> n.getFileName().toString()).filter(n -> n.endsWith(".bib"))
                     .filter(n -> n.startsWith("BibTeXML")).collect(Collectors.toList());
         }
     }
 
+    private static Path convertPackageNameToPath(String packageName) {
+        return Paths.get(SRC_TEST_RESOURCES +packageName.replace('.','/')+"/");
+    }
```

Comment:
@Siedlerchr yeah you're right. This problem seems to be a very well known [intelliJ problem] (http://stackoverflow.com/questions/27046642/access-file-in-junit-test-in-gradle-environment) that occurs here as well. I've added a gradle task that copys the testfiles to where they should be.